apps/src/craft/craft.js

```diff
@@ -161,6 +176,42 @@ Craft.init = function (config) {
   Craft.level = config.level;
   Craft.skin = config.skin;
 
+  var levelTracks = [];
+  if (Craft.level.songs && musicMetadata) {
+    levelTracks = musicMetadata.filter(function(trackMetadata) {
+      return Craft.level.songs.indexOf(trackMetadata.name) !== -1;
+    });
+  }
+
+  Craft.musicController = new MusicController(
+      studioApp.cdoSounds,
+      (filename) => config.skin.assetUrl(`music/${filename}`),
+      levelTracks,
+      levelTracks.length > 1 ? 7500 : null
+  );
+  if (studioApp.cdoSounds && !studioApp.cdoSounds.isAudioUnlocked()) {
+    // Would use addClickTouchEvent, but iOS9 does not let you unlock audio
+    // on touchstart, only on touchend.
+    var removeEvent = dom.addMouseUpTouchEvent(document, function () {
+      studioApp.cdoSounds.unlockAudio();
+      removeEvent();
+    });
```

Comment:
This is another feature that would be nice to pull straight into soundEffects.js, now that I see it duplicated - but that may be risky at this point.